app/actions/restrictedMailActions.js

```diff
@@ -0,0 +1,50 @@
+// @flow
```

Comment:
This filename should probably have a capital letter for consistency? (you probably have to use git mv with some weird args to change case on macOS)